### PR TITLE
BBS Release v5.0.zh.md

### PR DESCRIPTION
**Pull request by angryxueqiu**
Chinese version for BBS Release v5.0

### CODE REVIEW

File: content/bbs/BBS Release v5.0.zh.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+bounty = 1
+date = "2017-12-18"
+categories = [
+    "Development Updates",
+]
++++
+
+天空币BBS v5.0终于发布了，我们这次做了很多的更新呢！
+
+## 简化客户端
```
Comment (jayphbee) [inline content/bbs/BBS Release v5.0.zh.md]: “thin client” 翻译为 “轻客户端”比较好, 下同

File: content/bbs/BBS Release v5.0.zh.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+在以前的版本中，只能（也应该）通过本地提供的Web用户界面访问/提交内容。由于这个网络接口（以及它所调用的API）可以直接控制节点本身，并且使用存储在节点（或服务器端）中的私钥对内容进行签名，因此将其暴露给公众是不合适,不安全的。
+
+因此，引入简化客户端的时候, 我们需要用户管理以及签名提交内容的过程，以便客户端的处理。这需要全新改进的内容提交端点（或多于一个端点），并对前端进行大量更改。
+有关新内容提交过程的详细信息可以在BBS Wiki: [github.com/skycoin/bbs/wiki/Content-Submission-Process](https://github.com/skycoin/bbs/wiki/Content-Submission-Process)上找到.
+ 
+目前，前端加载非常缓慢。这是由于使用 [GopherJS](https://github.com/gopherjs)来处理种子和公钥/私钥生成以及数据的签名和验证所致。在将来的版本中，我们将使用原生JavaScript库来提高性能。
+
+
+## 命令-行界面(Command-line Interface)
```
Comment (jayphbee) [inline content/bbs/BBS Release v5.0.zh.md]: command-line Interface 翻译为 “命令行接口”，下同

**Comment by jayphbee** [comment]
Please make some modifications

File: content/bbs/BBS Release v5.0.zh.md (hunk 2 above)
Comment (angryxueqiu) [reply content/bbs/BBS Release v5.0.zh.md]: 感謝你的意見, 我想請問一下"Interface" 是不是通常翻譯作"接口" 比較常見

Comment (angryxueqiu) [reply content/bbs/BBS Release v5.0.zh.md]: 因為我見有時有些人會用例如"用戶界面"之類的

File: content/bbs/BBS Release v5.0.zh.md (hunk 1 above)
Comment (angryxueqiu) [reply content/bbs/BBS Release v5.0.zh.md]: Ok, thanks :)

File: content/bbs/BBS Release v5.0.zh.md (hunk 2 above)
Comment (angryxueqiu) [reply content/bbs/BBS Release v5.0.zh.md]: Yes it was done. see #539

**Comment by angryxueqiu** [comment]
@jayphbee @gz-c 
modifications was done, please see #539

**Pull request closed without merging**